Packs/MailListener/Integrations/MailListenerV2/MailListenerV2.py

```diff
@@ -291,7 +291,7 @@ def generate_search_query(latest_created_time: Optional[datetime],
         # Removing Parenthesis and quotes
         messages_query = messages_query.strip('()').replace('"', '')
     # Creating a list of the OR query words
-    messages_query_list = messages_query.split() + ['SINCE', latest_created_time]  # type: ignore[list-item]
+    messages_query_list = messages_query.split() + ['SINCE', latest_created_time - timedelta(days=1)]  # type: ignore[list-item]
```

Comment:
latest_created_time can be none here too no? make is it a bit problematic to do `-` operation

Reply:
Same here